### DISCUSSION TITLE
wg-easy容器部署wireguard组网服务器的问题？

### DISCUSSION THREAD

**Discussion by stqyz** [1Panel-dev/1Panel#5786] [Q&A]
请教下，就是我在腾讯云上部署了一台VPS服务器，系统镜像安装的是带Docker环境的Ubuntu 22.04，在系统上面安装了1Panel的面板来管理这台VPS，另外还安装了一个叫做wg-easy的容器镜像，用它来作为wireguard组网的中继服务器。目前这个wireguard的组网是成功的，但是我发现了一个问题，就是所有安装了wireguard客户端的电脑，都能用虚拟内网IP地址ping到wireguard中继服务器的容器的虚拟内网IP，但是在VPS的Ubuntu系统下是ping不通这些客户端电脑的虚拟IP地址的，只能在1Panel面板中通过ssh连接wg-easy容器，在容器的ssh下才能ping通客户端电脑的虚拟内网IP地址。如果我想让我的VPS直接在Ubuntu系统下能直接ping通客户端电脑的虚拟IP地址，需要怎么设置？

**Comment by stqyz** [comment]
补充一下，我在1Panel面板的容器管理界面中 ，可以看到wg-easy容器使用的网络模式是桥接模式，是否可以把它改为其他模式来实现我需要的功能呢？

**Comment by wan92hen** [comment]
可以改为 主机/host 模式试下。
